pkgs/desktops/gnome/misc/gnome-epub-thumbnailer/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, lib
+, fetchurl
+, meson
+, ninja
+, pkg-config
+, gnome
+, gdk-pixbuf
+, glib-networking
+, libarchive
+, libxml2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-epub-thumbnailer";
+  version = "1.7";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/gnome-epub-thumbnailer/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "sha256-S7Ah++RCgNuY3xTBH6XkMgsWe4GpG9e6WGvqDE+il1I=";
+  };
+
+  nativeBuildInputs = [
+    libxml2
```

Comment:
Why libxml in `nativeBuildInputs`?

Reply:
Followed the lead of gnome-tweaks. Unsure if this was the correct approach.

https://github.com/NixOS/nixpkgs/blob/4db165bd2f32e1d346dee3bcd1fe583189e45613/pkgs/desktops/gnome/misc/gnome-tweaks/default.nix#L36

Reply:
If a dependency’s code is expected to run during build-time, it goes to `nativeBuildIputs`. If the code is expected to run at run-time (on host platform, to use Autotools terminology), it goes to `buildInputs`.

`libxml2` contains executable programs which tend to be used at build-time but it also contains a library, which is typically used at run-time. Looking at the `meson.build` file, it looks like here the latter is the case.